sklearn/metrics/tests/test_classification.py

```diff
@@ -1374,7 +1374,27 @@ def test_log_loss():
     loss = log_loss(y_true, y_pred)
     assert_almost_equal(loss, 1.0383217, decimal=6)
 
+    #test labels option
+
+    X = [[1,1], [1,1], [2,2], [2,2]]
+    y_label = [1,1,2,2]
+
+    X_test = [[2,2], [2,2]]
+    y_true = [2,2]
+    y_score = np.array([[0.1,0.9], [0.1, 0.9]])
+    
+    # because y_true label are the same, if not use labels option, will get error
```

Comment:
Should we just raise a helpful error for this case, advising to use the `labels` option?